src/lib/Scenes/Artwork/Components/ImageCarousel/ImageCarousel.tsx

```diff
@@ -96,6 +91,17 @@ export const ImageCarouselFragmentContainer = createFragmentContainer(ImageCarou
       image_url: imageURL
       width
       height
+      deepZoom {
+        Image {
+          TileSize
+          Url
+          Format
+          Size {
+            Width
+            Height
+          }
+        }
+      }
```

Comment:
Errrr, what’s with the naming of these fields?? We should fix these to be `camelCase`.

Reply:
i aliased the fields to be lower-cased so we can fix this more easily later